.vscode/launch.json

```diff
@@ -8,120 +8,51 @@
-            "console": "integratedTerminal",
-            "preLaunchTask": "dapr-SeatAdjuster-run",
-            "postDebugTask": "dapr-SeatAdjuster-stop",
-            "env": {
-                "SDV_MIDDLEWARE_TYPE": "dapr",
-                "DAPR_GRPC_PORT": "50001",
-                "DAPR_HTTP_PORT": "3500",
-                "SEATSERVICE_DAPR_APP_ID": "seatservice",
-                "VEHICLEDATABROKER_DAPR_APP_ID": "vehicledatabroker"
-            }
-        },
-        {
-            "type": "python",
-            "request": "launch",
-            "name": "examples/seat-adjuster (native)",
-            "cwd": "${workspaceFolder}/examples/seat-adjuster",
-            "program": "${workspaceFolder}/examples/seat-adjuster/src/main.py",
-            "console": "integratedTerminal",
-            "env": {
-                "SDV_MIDDLEWARE_TYPE": "native",
-                "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
-                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555",
-                "SDV_SEATSERVICE_ADDRESS": "grpc://localhost:50051"
-            }
-        },
-        {
-            "type": "python",
-            "request": "launch",
-            "name": "examples/array-datatype (native)",
-            "cwd": "${workspaceFolder}/examples/array-datatype",
-            "program": "${workspaceFolder}/examples/array-datatype/src/main.py",
+            "name": "debug example (native middleware)",
+            "cwd": "${workspaceFolder}/examples/${input:example}",
+            "program": "${workspaceFolder}/examples/${input:example}/src/main.py",
             "console": "integratedTerminal",
             "env": {
                 "SDV_MIDDLEWARE_TYPE": "native",
                 "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
+                "SDV_HVACSERVICE_ADDRESS": "grpc://localhost:50052",
+                "SDV_SEATSERVICE_ADDRESS": "grpc://localhost:50051",
                 "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555"
             }
         },
         {
             "type": "python",
             "request": "launch",
-            "name": "examples/vdb-queries (native)",
-            "cwd": "${workspaceFolder}/examples/vdb-queries",
-            "program": "${workspaceFolder}/examples/vdb-queries/src/main.py",
-            "console": "integratedTerminal",
-            "env": {
-                "SDV_MIDDLEWARE_TYPE": "native",
-                "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
-                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555",
-                "SDV_SEATSERVICE_ADDRESS": "grpc://localhost:50051"
-            }
-        },
-        {
-            "type": "python",
-            "request": "launch",
-            "name": "examples/static-rule (native)",
-            "cwd": "${workspaceFolder}/examples/static-rule",
-            "program": "${workspaceFolder}/examples/static-rule/src/main.py",
+            "name": "debug example (Dapr middleware)",
```

Comment:
Why are we not using just one entry and take the middleware from the input?

Reply:
I agree to Dominic. Nice "side effect" is that you can "pick" the middleware once and then only have to choose the example.

Reply:
You still have to pick the middleware on every run, since you have to select the correct example. If you want to keep this than we should maybe at least make the middleware more prominent in the selection like 
```suggestion
            "name": "Dapr: debug example",
```

Reply:
Maybe that's a difference between MacOS and Linux? In VSCode under Linux you choose the middleware once:
![grafik](https://user-images.githubusercontent.com/43466862/237029288-08681292-d598-4131-b0f9-c38043f62581.png)
And then press F5 (or klick the "play button") and just have to choose the example, but not the MW again ...

Reply:
Simplified the launch config "titles"